src/blog/2023/06/flowforge-1-8-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: FlowForge now offers High Availability Node-RED
+subtitle: FlowForge 1.8 makes Node-RED applications more reliable and scalable, plus more streamline deployment pipelines.
+description: FlowForge now supports High Availibility, DevOps Pipelines and more
+date: 2023-06-08 
+authors: ["marian-demme"]
+image: /blog/2023/06/images/release-1.8.0.png
+---
+
+The FlowForge 1.8 update introduces a suite of features designed to optimize your Node-RED experience. High Availability ensures your business-critical processes stay running under heavy workloads, while DevOps Pipelines allow for smoother, staged deployments. Additionally, the integration of a new User Interface for the Device Agent significantly streamlines the configuration process.
```

Comment:
I think this section needs to be improved. I think it under sells the 1.8 features.  I would suggest something like the following:

FlowForge 1.8 introduces two key features that allow organizations to reliably deploy Node-RED applications into production. In 1.8, it is now possible to run Node-RED applications with high availability so the application is more scalable and more fault tolerant. FlowForge 1.8 also introduces software deliver pipelines so development teams can now setup dev/test/production environments for their Node-RED applications.

Reply:
> It is now possible to run Node-RED applications with high availability so the application is more scalable and more fault tolerant.

Less comma's should be better

Reply:
```suggestion
FlowForge 1.8 introduces two key features that allow organizations to reliably deploy Node-RED applications into production. In 1.8, it is now possible to run Node-RED applications with high availability so the application is more scalable and more fault tolerant. FlowForge 1.8 also introduces software deliver pipelines, so development teams can now set up dev/test/production environments for their Node-RED applications.
```